pkg/discovery/discovery.go

```diff
@@ -51,35 +61,33 @@ func Run(kubeClient kubernetes.Interface, cfg *config.Config) []error {
 		}
 	}
 
-	// closure used to collect and report errors.
-	rollup := func(err []error) {
-		if err != nil {
-			errlst = append(errlst, err...)
-		}
-	}
-
 	// 4. Run the plugin aggregator
-	errlst = append(errlst, pluginaggregation.Run(kubeClient, cfg.LoadedPlugins, cfg.Aggregation, outpath)...)
+	trackErrorsFor("running plugins")(
```

Comment:
Should the plugins.Run just log errors inline and not return?  This was my conundrum for *.  Reporting up to this layer didn't make as much sense as we had originally thought.